index.rst

```diff
@@ -27,21 +27,25 @@ summarized below and in :ref:`Figure 1<fig-nightly-proc>`, the first portion of
    :alt: First portion of the LDM-151 Nightly Overview flowchart, from ISR to Image Differencing
    
    The first portion of the Nightly Processing Pipeline overview, from Figure 1 of 
-   LDM-151 [swi17]_. This encompasses all of "Single Frame Processing" and part of "Alert Detection."
+   :cite:`LDM-151`. This encompasses all of "Single Frame Processing" and part of "Alert Detection."
 
 A number of packages in the LSST Software Stack have been developed to perform
 portions of one or more of these four tasks, but they are not yet linked together
 in an automated way. This Prototype AP Pipeline is the first step toward the goal
 of an automatic Level 1 Pipeline that can process a set of raw images through 
 each of these steps and verify that the output is what we expect.
 
+To achieve this, there are two main python modules under development: 
+`ap_pipe <https://github.com/lsst-dm/ap_pipe>`_ and `ap_verify <https://github.com/lsst-dm/ap_verify>`_.
+The former is described here, and is used by the latter.
+
 
 .. _Dataset:
 
 Dataset
 =======
 
-We use images from the High Cadence Transient Survey (HiTS, [for16]_), a three-year survey 
+We use images from the High Cadence Transient Survey (HiTS) :cite:`for16`, a three-year survey 
```

Comment:
Did the technote powers that be say anything about moving references into central bibliography files? If you're going to have to do that later, than an ADS ID might be a more portable name than `for16` (but beware `...` in IDs).

Reply:
The short answer is, I don't believe there is a standard, documented way for handling this yet. Good point to keep in mind once there is, though!